### DISCUSSION TITLE
Welcome to canvas-editor Discussions!

### DISCUSSION THREAD

**Discussion by Hufe921** [Hufe921/canvas-editor#546] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by yaojianchao0223** [comment]
在使用组件时发现，在编辑文档发现的问题，对于文档，它的数据结构是根据样式或者结构等不固定分块的，如果我要在第二段插入内容，可以使用api获取到光标位置，包括index,rowindex等，但是是随机分块的，就无法将内容插入到指定内容之后的数组结构中对应的数组中，除非往对应的内容往后拼接，这样就会使用到分块的样式结构，有什么好的处理方式，支持往数组指定的下标中加入新增的结构数据吗？
